ethereum/gnosis_protocol_v2/view_solvers.sql

```diff
@@ -20,75 +20,15 @@ registered_solvers as (
-                 ('0x70f3c870b6e7e1d566e40c41e2e3d6e895fcee23', 'barn', 'QuasiModo'),
-                 ('0x97dd6a023b06ba4722aF8Af775ec3C2361e66684', 'barn', '0x'),
-                 ('0x6372bcbf66656e91b9213b61d861b5e815296207', 'barn', 'ParaSwap'),
-                 ('0x158261d17d2983b2cbaecc1ae71a34617ae4cb16', 'barn', 'MIP'),
-                 ('0x8c9d33828dace1eb9fc533ffde88c4a9db115061', 'barn', '1inch'),
-                 ('0xbfaf2b5e351586551d8bf461ba5b2b5455b173da', 'barn', 'Baseline'),
-                 ('0xb8650702412d0aa7f01f6bee70335a18d6a78e77', 'barn', 'Naive'),
-                 ('0x583cD88b9D7926357FE6bddF0E8950557fcDA0Ca', 'barn', 'DexCowAgg'),
-                 ('0x6c2999b6b1fad608ecea71b926d68ee6c62beef8', 'barn', 'Legacy'),
-                 ('0xED94b86275447e28ddbDd17BBEB1f62D607b5119', 'barn', 'Legacy'),
-                 ('0x8ccc61dba297833dbe5d95fd6360f106b9a7576e', 'barn', 'Naive'),
-                 ('0x0d2584da2f637805071f184bcfa1268ebae8a24a', 'barn', 'Baseline'),
-                 ('0xa0044c620da7f2876da7004719b8370eb7be5e50', 'barn', 'MIP'),
-                 ('0xda324c2f06d3544e7965767ce9ca536dcb67a660', 'barn', 'QuasiModo'),
-                 ('0xe33062a24149f7801a48b2675ed5111d3278f0f5', 'barn', '1inch'),
-                 ('0x080a8b1e2f3695e179453c5e617b72a381be44b9', 'barn', 'ParaSwap'),
-                 ('0xde786877a10dbb7eba25a4da65aecf47654f08ab', 'barn', '0x'),
-                 ('0xdae69affe582d36f330ee1145995a53fab670962', 'barn', 'DexCowAgg'),
-                 ('0x0b78e29ee55aa73b366730cf512d65c514eeb196', 'barn', 'BalancerSOR'),
-                 ('0x22dee0935c77d32c7241362b14e76fc2d5ef657d', 'barn', 'BalancerSOR'),
-                 ('0x5b0bfe439ab45a4f002c259b1654ed21c9a42d69', 'barn', 'PLM'),
-                 ('0x0798540ee03a8c2e68cef19c56d1faa86271d5cf', 'service', 'Withdraw'),
-                 ('0xdf30c9502eafea21ecc8410108dda338dd5047c5', 'service', 'Withdraw'),
-                 ('0x256bb5ad3dbdf61ae08d7cbc0b9223ccb1c60aae', 'service', 'Withdraw'),
-                 ('0x84e5c8518c248de590d5302fd7c32d2ae6b0123c', 'service', 'Withdraw'),
-                 ('0xa03be496e67ec29bc62f01a428683d7f9c204930', 'service', 'Withdraw'),
-                 ('0x2caef7f0ee82fb0abf1ab0dcd3a093803002e705', 'test', 'Test Solver 1'),
-                 ('0x56d4ed5e49539ebb1366c7d6b8f2530f1e4fe753', 'test', 'Test Solver 2')
-         ) as _
 )
 -- Combining the metadata with current activation status for final table
 select solver as address,
        case when environment is not null then environment else 'new' end as environment,
        case when name is not null then name else 'Uncatalogued' end      as name,
        active
 from registered_solvers
-         left outer join known_solver_metadata
-                         on solver = replace(address, '0x', '\x')::bytea;
+         left outer join gnosis_protocol_v2.solver_names
+                         on solver = replace(address::text, '0x', '\x')::bytea;
```

Comment:
I had actually forgotten the original reason I had this. Actually the CSV itself would be more readable if it had 0x addresses. Perhaps I should revert that and make the named solver table with strings. Although I think the effort here is probably more than anyone wants to deal with.

Reply:
for updating solver list: yes, i believe an initial PR on just that file to update, merge, then refer in next PR may be best option. the alternative would require us to alter the URL reference each time. in our scenario, the URL can remain static, as we merge into master first.

feel free to proceed as you see fit on the second point about readability.

Reply:
Not so sure I'm keen on having to make two PRs, just to update the solver names... maybe we should forget this whole idea. It was supposed to make things cleaner.

Reply:
yeah, the other option would be to include csv file in same PR, then a dune teammate would need to determine the branch used and alter the URL link prior to running the command. for example, this URL would be:
https://raw.githubusercontent.com/duneanalytics/abstractions/d16117c907fe67e57c0db1eddc3efe554aa74461/ethereum/gnosis_protocol_v2/solver_names.csv

'd16117c907fe67e57c0db1eddc3efe554aa74461' would be specific to this PR.

if you want to abandon the idea, feel free to close this, then we can revisit again if ideas surface

Reply:
Yes I think I may abandon this. Sorry to waste your time here. Will close it tomorrow. Or turn it into a revert of the previous one.

Reply:
and I apologize in advance for the stupidity